float64 des_motor_torque_left_A
float64 des_motor_torque_left_B
float64 des_motor_torque_left_hip
float64 des_motor_torque_right_A
float64 des_motor_torque_right_B
float64 des_motor_torque_right_hip

bool dutyCycleTest
float64 dc_ip
float64 dc_ic
float64 dc_tp
float64 dc_tc

bool limitCurrent

